docs/handbook/systems-administration.md

```diff
@@ -1081,13 +1139,9 @@ IF you cannot ping an external IP (e.g. google.com) run this command and try aga
 credit for this section of docs go to [/u/127b](https://www.reddit.com/user/127b)
 
 
-### Automatic Configuration (NWAM)
-
-Network Auto-Magic (NWAM) is a new approach to managing network interfaces that was introduced with OpenSolaris.
-NWAM introduced network profiles to be able to change network settings on the fly.
-One important reason to redesign networking was the increasing importance of wireless networking and the need to cope with its dynamic nature.
+### More on Automatic Configuration (NWAM)
 
-While usually server and desktop installations tend to use default network configurations, laptop users can leverage nwam network configuations.
+The following is a more in depth discussion of NWAM, which normally works completely automatic, but can be customized.
```

Comment:
completely automatic --> completely automatically
or possibly
 -> completely automatically without any user interaction, but user interaction to customize is also possible.